docs/modules/ROOT/pages/getting-started.adoc

```diff
@@ -22,7 +22,8 @@ Then install K8up itself:
 
 [source,bash]
 ----
-helm install appuio/k8up
+kubectl create namespace k8up-operator
+helm install appuio/k8up --namespace k8up-operator --generate-name
```

Comment:
I'd suggest to rather specify a name than generating one
```suggestion
helm install k8up appuio/k8up --namespace k8up-operator
```

Reply:
Sounds good to me